src/main/kotlin/omok/model/external/rule/BlackRenjuRule.kt

```diff
@@ -0,0 +1,156 @@
+package omok.model.external.rule
+
+import omok.model.external.rule.type.Foul
+import omok.model.external.rule.type.Violation
+import omok.model.external.rule.type.Violation.Companion.FOUL_CONDITION_SIZE
+import omok.model.external.rule.type.Violation.Companion.MAX_EMPTY_SIZE
+import omok.model.external.rule.type.Violation.Companion.OVERLINE_SIZE
+import omok.model.external.rule.type.WhiteBlocked
+import omok.model.external.rule.wrapper.direction.Directions
+import omok.model.external.rule.wrapper.point.Point
+
+class BlackRenjuRule(
+    boardWidth: Int = DEFAULT_BOARD_WIDTH,
+    boardHeight: Int = DEFAULT_BOARD_HEIGHT,
+) : OmokRule(boardWidth, boardHeight) {
+
+    override fun checkDoubleFoul(
+        blackPoints: List<Point>,
+        whitePoints: List<Point>,
+        startPoint: Point,
+        foul: Foul,
+    ): Violation = checkFoulByAllDirections(blackPoints, whitePoints, startPoint, foul)
+
+    override fun checkOverline(
+        stonesPoints: List<Point>,
+        startPoint: Point,
+    ): Violation {
+        if (checkSerialSameStonesBiDirection(stonesPoints, startPoint, OVERLINE_SIZE)) return Violation.OVERLINE
```

Comment:
```suggestion
        if (checkSerialSameStonesBiDirection(stonesPoints, startPoint, Violation.OVERLINE_SIZE)) return Violation.OVERLINE
```
companion object에 정의된 상수값을 import를 하게 되면 코드상으로는 간단하게 작성할 수 있지만, 어떤 클래스에서 정의된 상수인지 알기가 어렵습니다.
import를 하기보다는 명시적으로 작성해보면 어떨까요?